### ISSUE TITLE
Show example of ESP32 + WHIP in examples

### ISSUE THREAD

**Issue by Sean-Der** [sepfy/libpeer#139]
@sepfy would you be ok if I modified examples/esp32 to allow users to do WHIP with it? Make make it something users can configure in `idf.py menuconfig` ?

Would you also mind tagging a new release? I am going to write a blogpost of how to use WHIP + libpeer against Pion and v0.0.2 doesn't have the fixes to work with Pion.

thank you!

**Comment by sepfy** [comment]
Of course. Let me know which commit you'd like to use as the first release for your blog post.

**Comment by Sean-Der** [comment]
Should I make `examples/esp32-whip` or add WHIP options to `examples/esp32`?

I think adding WHIP to `examples/esp32` is better. I want to implement it however you think is best though!

**Comment by sepfy** [comment]
yes. adding to `examples/esp32` is great. Thanks!
